List5_2.sc

```diff
@@ -0,0 +1,86 @@
+sealed trait BT[+A]
+case object Empty extends BT[Nothing]
+case class Node[+A](elem:Int, left:BT[A], right:BT[A]) extends BT[A]
+
+import scala.math.pow
+
+//Zadanie 1
+def createTree2[A](depth : Int) : BT[Int] = {
```

Comment:
Skąd createTree2? Skąd 2 w nazwie? Coś oznacza czy śmieć roboczy?

Reply:
Już poprawione wszystko, tak, był to śmieć roboczy